templates/pages/links.html

```diff
@@ -8,6 +8,13 @@
 		<p>We gather useful links to tutorials, good practices, etc. </p>
 
 	</div>
-
+<div>
+    <p><a href="http://www.designforfounders.com/web-app-ux/">15 Timeless Rules for Creating Intuitive Web Apps (With Examples)</a></p>
```

Comment:
Please use tabs for indentation, not spaces (this is something you can set in your text editor). Again, it's not something that affects functionality, but it's good practice to maintain the same style across all files in a project.